cdk/src/stack.ts

```diff
@@ -45,42 +47,57 @@ export class HeyAmplifyStack extends Stack {
     Tags.of(this).add('app:name', this.appName)
     Tags.of(this).add('app:env', this.envName)
 
-    const vpc = new ec2.Vpc(this, `vpc`, {
+    const vpc = new ec2.Vpc(this, `Vpc`, {
       maxAzs: 2,
-      vpcName: `${this.appName} VPC`,
+      vpcName: `vpc-${this.appName}-${this.envName}`,
     })
 
-    const cluster = new ecs.Cluster(this, `fargate-cluster`, {
+    const cluster = new ecs.Cluster(this, `Cluster`, {
       vpc,
       containerInsights: true,
       enableFargateCapacityProviders: true,
     })
 
-    const filesystem = new efs.FileSystem(this, 'filesystem', {
+    const filesystem = new efs.FileSystem(this, 'Filesystem', {
       vpc,
-      vpcSubnets: {
-        subnetType: ec2.SubnetType.PUBLIC,
-        onePerAz: true,
-      },
     })
 
-    new HeyAmplifyApp(this, `bot`, {
+    let subdomain: AmplifyAwsSubdomain | undefined
+    if (props.subdomain) {
+      subdomain = new AmplifyAwsSubdomain(this, 'Subdomain', props.subdomain)
+    }
+
+    // create bucket for SQLite backups with Litestream
+    const bucket = new s3.Bucket(this, 'Bucket', {
+      bucketName: `${this.appName}-${this.envName}-bucket`,
+      // if env is destroyed, empty & remove bucket
+      removalPolicy: RemovalPolicy.DESTROY,
+      autoDeleteObjects: true,
+    })
+
+    new HeyAmplifyApp(this, `Bot`, {
+      bucket,
       cluster,
       docker: {
         name: `${this.appName}-bot`,
         context: PROJECT_ROOT,
         dockerfile: 'Dockerfile',
         environment: {
-          DATABASE_URL: `file:../db/${this.envName}.db`,
+          DATABASE_URL: `file:/data/${this.envName}.db`,
           ...getSvelteKitEnvironmentVariables(this.envName),
         },
       },
       secrets,
-      subdomain: props.subdomain
-        ? new AmplifyAwsSubdomain(this, 'subdomain', props.subdomain)
-        : undefined,
+      subdomain,
       filesystem,
-      filesystemMountPoint: '/usr/src/db',
+      filesystemMountPoint: '/data',
```

Comment:
Looks like `filesystemMountPoint` and `DATABASE_URL` could potentially get out of sync which would affect the litestream container which uses both vars. Should they be related somehow?

Reply:
Great callout. I've refactored this to set `filesystemMountPoint` outside of this config and construct the `DATABASE_URL` env var using this variable

```ts
DATABASE_URL: `file:${filesystemMountPoint}/${this.envName}.db`,
```